crates/sui-types/src/sui_system_state.rs

```diff
@@ -553,6 +581,105 @@ impl Default for SuiSystemState {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq, JsonSchema)]
+pub enum SuiSystemState {
+    V1(SuiSystemStateInnerV1),
+}
```

Comment:
lets use adjacent tag for enums
https://serde.rs/enum-representations.html#adjacently-tagged